code/modules/projectiles/guns/projectile/pistol/mandella.dm

```diff
@@ -2,6 +2,7 @@
 	name = "FS HG .25 CS \"Mandella\""
 	desc = "A rugged, robust operator handgun with inbuilt silencer. Chambered in rifle caseless ammunition, this time-tested handgun is \
 			your absolute choise if you need to take someone down silently, as it's deadly, produces no sound and leaves no traces. \
+			Build to enhance armor penetration abilities of ammo. \
```

Comment:
"Built to have higher penetration capabilities than its peers." or just "Built to have enhanced armor penetration abilities." or just replace Build with Built I guess.